packages/flutter/lib/src/painting/decoration_image.dart

```diff
@@ -189,7 +239,13 @@ class DecorationImage {
         '$repeat',
       if (matchTextDirection)
         'match text direction',
-      'scale: $scale',
+      'scale $scale',
+      'opacity $opacity',
+      '$filterQuality',
+      if (invertColors)
+        'invert colors',
```

Comment:
I wonder if this should say `invertColors: true` 🤔

Reply:
I followed the style above with matchTextDirection. Should we change that for both or just for inverting colors, same for `isAntiAlias`?

Reply:
Oh yes, so you did. Never mind then, don't bother with that change. It was
a bad idea on my account!